csharp/src/IceBox/msbuild/icebox/icebox.csproj

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="../../../../msbuild/ice.common.props" />
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <AssemblyName>iceboxnet</AssemblyName>
+    <Version>$(IceVersion)</Version>
+    <OutputPath>../../../../bin</OutputPath>
```

Comment:
Can we use the same iceboxnet for .NET and .NET Framework?

I thought we would need separate builds.

Reply:
we build two versions just using the same project